src/content/blog/the-three-scales-of-linting/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+description: "TODO"
+image:
+  alt: "TODO"
+  src: "~/assets/blog/zootopia-flash-mid-smile.png"
+pubDate: 2025-07-08
+series: Linters
+title: "The Three Scales of Linting"
+---
+
+import LabeledImage from "~/components/blog/mdx/LabeledImage.astro";
+import LabeledVideo from "~/components/blog/mdx/LabeledVideo.astro";
+
+There's been a lot of movement around linting and performance in web development recently.
+Ultra-fast native speed linters such as Biome, deno lint, and Oxlint paved the way for shockingly fast speedups -- at the cost of switching languages.
+TypeScript-native projects such as tsl, TSSLint, and my own Flint are working on integrating more closely with TypeScript to attain some of those performance benefits while retaining the familiarity of TypeScript.
```

Comment:
TODO: this is too wordy; maybe move all but the first sentence or so into the closing paragraph?